excercsises/3_synchronization/contact.spec.js

```diff
@@ -1,17 +1,30 @@
 var contactPage = require('./pages/contact');
 
-describe('Protractor Workshop app', function() {
+describe('Protractor Workshop app', function () {
 
 	beforeEach(function () {
 		contactPage.get();
 	});
 
-	it('hould have Contact page with title "Protractor workshop | Contact us"', function(){
-		expect(contactPage.getTitle()).toEqual("Protractor workshop | Home");
+	it('hould have Contact page with title "Protractor workshop | Contact us"', function () {
+		expect(contactPage.getTitle()).toEqual("Protractor workshop | Contact us");
 	});
 
-	xit('should display text "Your message has been sent." when user sends message  ', function(){
-		
+	it('should display text "Your message has been sent." when user sends message  ', function () {
```

Comment:
Please use pages in ex 3. also use expected conditions to wait for lastMessage to be displayed